pkg/controller/.import-restrictions

```diff
@@ -270,7 +270,6 @@
         "k8s.io/kubernetes/pkg/security/apparmor",
         "k8s.io/kubernetes/pkg/util/parsers",
         "k8s.io/kubernetes/pkg/fieldpath",
-        "k8s.io/kubernetes/pkg/scheduler/volumebinder",
```

Comment:
please revert this line, https://github.com/kubernetes/kubernetes/pull/83526 is in flight and will do this cleanup.

Reply:
he would update soon.

Reply:
https://github.com/kubernetes/kubernetes/pull/83526 has been merged after the above review.
I guess this PR should be updated based on the latest one, right?